job_definitions/notify_suppliers_of_framework_application_events.yml

```diff
@@ -1,15 +1,4 @@
 {% set environments = ['preview', 'production'] %}
-{% set notify_template_map = {
-    '21_new_clarification_questions_answers': 'ed9944eb-112e-4f04-a4bb-19f397550ffe',
-    '22_clarification_questions_close_next_week': '119dde9a-1e44-4b1d-9224-2f7020a74f9f',
-    '23_clarification_questions_close_today': '1464000b-1027-4fa5-97fe-7280ab9daf1d',
-    '24_clarification_questions_closed': '8f98a25f-0a9e-4a96-9034-2ffc6b91cedc',
-    '26_applications_close_soon': 'efec07be-a914-4b50-b452-48563888930e',
-    '27_applications_close_today': '441142fb-adaf-4510-b1f4-e3f918993881',
-    '40_standstill_starts_today': '8c5b822a-844c-4376-8a76-4bb3706a5e35',
-    '42_standstill_has_ended': '395253ed-1ab6-424e-b032-1f197d9271e7',
-  }
```

Comment:
I'm a bit baffled why this didn't work - we use dicts elsewhere with Jenkins job templates.

Perhaps it needs to be `notify_template_map.EMAIL_TYPE` instead of `notify_template_map[EMAIL_TYPE]`?

Reply:
The problem is that `EMAIL_TYPE` isn't set until you run the job on Jenkins, but Jinja compiles the template when you push with Ansible

Reply:
🤦‍♀ Of course. OK!